packages/client-core/src/admin/components/Instance/PatchGameserver.tsx

```diff
@@ -0,0 +1,113 @@
+import Button from '@mui/material/Button'
+import Container from '@mui/material/Container'
+import DialogActions from '@mui/material/DialogActions'
+import DialogTitle from '@mui/material/DialogTitle'
+import Drawer from '@mui/material/Drawer'
+import FormControl from '@mui/material/FormControl'
+import MenuItem from '@mui/material/MenuItem'
+import Paper from '@mui/material/Paper'
+import Select from '@mui/material/Select'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useAuthState } from '../../../user/services/AuthService'
+import { useFetchAdminLocations } from '../../common/hooks/Location.hooks'
+import { GameserverService } from '../../services/GameserverService'
+import { LocationService, useLocationState } from '../../services/LocationService'
+import { useStyles } from '../../styles/ui'
+
+interface Props {
+  open: boolean
+  handleClose: any
+  closeViewModel?: any
+}
+
+const PatchGameserver = (props: Props) => {
+  const { open, handleClose, closeViewModel } = props
+  const classes = useStyles()
+  const [state, setState] = React.useState({
+    location: '',
+    locationError: ''
+  })
+
+  const { t } = useTranslation()
+  const authState = useAuthState()
+  const user = authState.user
+  const adminLocationState = useLocationState()
+  const location = adminLocationState
+  const adminLocations = adminLocationState.locations
+
+  useFetchAdminLocations(user, adminLocationState, LocationService)
+
+  React.useEffect(() => {
+    if (location.created.value) {
+      closeViewModel(false)
+      setState({
+        ...state,
+        location: ''
+      })
+    }
+  }, [location.created.value])
```

Comment:
with new changes which @speigg mentioned in discord the .value isn't required here anymore.